### DISCUSSION TITLE
how to have multiple profiles

### DISCUSSION THREAD

**Discussion by nattgamer** [kaliiiiiiiiii/Selenium-Profiles#76] [Q&A] [answered]
profile = profiles.Windows() will access and extract unique data, how to get more data to change browser?

**Comment by ttlns** [comment, marked as answer]
not exactly, `profiles.Windows()` and `profiles.Android()` are predefined profiles. 
For "cloning" a browser, you can go to [js.do/kaliiiiiiiiiii/get_profile](https://js.do/kaliiiiiiiiiii/get_profile) on that device.

**Comment by ttlns** [comment]
Closing as it seems to be resolved
